draft-ietf-mops-streaming-opcons.md

```diff
@@ -331,15 +331,14 @@ important to consider the effects of network design decisions
 on application-level performance, with considerations for
 the impact on video delivery.
 
-This document aims to provide a taxonomy of networking issues as
```

Comment:
Hmm, I had a comment here that seems to have been lost, not sure why.

Anyway, I said: +1 on removing 'taxonomy', but I'm less enthused about removing "networking issues as they relate to quality of experience in internet video delivery."

What do you think of "This document examines networking issues as they relate to quality of experience in internet video delivery.  The focus is on capturing..."?

Reply:
Good call. I adopted this without changes.